Cargo.toml

```diff
@@ -35,11 +36,14 @@ libc = { version = "0.2.58", optional = true }
 syslog-3 = ["syslog3"]
 syslog-4 = ["syslog4"]
 syslog-6 = ["syslog6"]
+syslog-7 = ["syslog7"]
 reopen-03 = ["reopen03", "libc"]
 reopen-1 = ["reopen1", "libc"]
 meta-logging-in-format = []
 date-based = ["chrono"]
 
+# default = ["syslog-7"]
```

Comment:
commented config option - could you remove this?

Reply:
Ah, sure, that was for testing

Reply:
Done.